fdbclient/ReadYourWrites.actor.cpp

```diff
@@ -1440,10 +1448,10 @@ void ReadYourWritesTransaction::writeRangeToNativeTransaction( KeyRangeRef const
 				switch(op[i].type) {
 					case MutationRef::SetValue:
 						if (op[i].value.present()) {
-							tr.set( it.beginKey().assertRef(), op[i].value.get(), false );
-						} else {
-							tr.clear( it.beginKey().assertRef(), false );
-						}
+						    tr.set(it.beginKey().assertRef(), op[i].value.get(), false);
```

Comment:
Something has happened with tabs/spaces starting from here for the next 20 lines or so.